lib/egl/egl_main.cpp

```diff
@@ -437,27 +424,24 @@ EglMainThread::Handle EglMainThread::CreateWindow(EglWindow *caller, int w,
    }
 #endif //GLVIS_USE_EGL
 #ifdef GLVIS_USE_CGL
-   if (!out_hnd.isInitialized())
-   {
-      return out_hnd;
+   if (!out_hnd.isInitialized()) { return out_hnd; }
+   CGLError err = CGLSetCurrentContext(out_hnd.ctx.get());
+   if (err != kCGLNoError) { return out_hnd; }
+   // initialize GLEW before using any OpenGL functions
+   const auto status = caller->initGLEW(legacy_gl);
+   MFEM_VERIFY(status, "GLEW initialization failed!");
+   assert(glGetError() == GL_NO_ERROR);
+   const auto *renderer = glGetString(GL_RENDERER);
+   if (strstr((const char*)renderer, "Software") != nullptr)
+   {
+      PRINT_DEBUG("Using software renderer; expect limited functionality");
```

Comment:
I am just curious, does this happen often on Mac? 🤔

Reply:
never - I removed it in https://github.com/GLVis/glvis/pull/345/commits/e873d0c5c2524b4b1fb6bdc0669970a323cbfd5a